sources/packages/backend/apps/api/src/services/institution/institution.service.ts

```diff
@@ -705,6 +708,16 @@ export class InstitutionService extends RecordDataModelService<Institution> {
     return INSTITUTION_TYPE_BC_PRIVATE === institutionType.institutionType.id;
   }
 
+  /**
+   * Check if institution type is BC public.
+   * @param institutionId institution id.
+   * @returns whether the institution type is public.
+   */
+  async isBCPublicInstitution(institutionId: number): Promise<boolean> {
```

Comment:
Just a suggestion, we already have a method to check if the institution is private, which has one reference. Can we just create a single method that return either the institution is INSTITUTION_TYPE_BC_PRIVATE or INSTITUTION_TYPE_BC_PUBLIC?. Thoughts @ann-aot @andrewsignori-aot @sh16011993 @dheepak-aot

Reply:
I see the point @guru-aot.
Maybe we could create read-only properties a the entity model that will do the check based on the constants ` INSTITUTION_TYPE_BC_PRIVATE ` and `INSTITUTION_TYPE_BC_PUBLIC`.
IMO this is something that can be considered an entity-level concern. Would be like an emulation of a DB computed property.

Reply:
I agree that it should just be a entity level concern.

Reply:
@guru-aot I get what you are saying, but to have a single method returning private or public, we need to have a thrid category to pull all these institution types.

![image](https://user-images.githubusercontent.com/54600590/230184295-407a925a-f74b-4bad-8f82-de3e5941b6cf.png)

Reply:
I get the point team, I was just talking about combining 2 methods and returning a single value either public or private. I guess its going way out of the scope of this PR. Pausing the discussion for now.

Reply:
@guru-aot I am exactly trying to comprehend your idea. for e.g. if there is a method called `checkIfPrivateOrPublic(): Private | Public`, what will it return if the institution type is International? let me know if I am missing somehthing.

Reply:
@guru-aot feel free to pause the discussion, from your end, please leave the comment open.
@andrepestana-aot I believe that this method will no longer be needed because we will build the decorator in the future.
If it is still needed let's move it to the entity model alongside the `isBCPrivate`.

Reply:
I removed the method isBCPublicInstitution